spec/features/indexing_custom_document_spec.rb

```diff
@@ -92,14 +91,12 @@
 
       it 'matches EAD data' do
         expect(doc['repository_ssm'].first).to eq '1118 Badger Vine Special Collections'
-        expect(doc['repository_sim'].first).to eq '1118 Badger Vine Special Collections'
```

Comment:
I was curious about this too.  It looks like the expectation is an exact duplicate of the one above, or is there something I'm not seeing?